snap/snapcraft.yaml

```diff
@@ -0,0 +1,50 @@
+name: foundationdb
+version: 6.0.18
+summary:  An open source, distributed, transactional key-value store.
+description:| 
+A multi-model NoSQL database.
+
+grade: stable
+confinement: classic
+base: core18
+
+architectures:
+  - build-on: amd64
+
+apps:
+  foundationdb:
+    command: bin/desktop-launch $SNAP/bin/FoundationDB
+    environment:
+      LD_LIBRARY_PATH: "$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pulseaudio:$LD_LIBRARY_PATH"
```

Comment:
Why is pulse audio required?

Reply:
No, I thought I had removed it. I just fixed it now